package.json

```diff
@@ -98,12 +94,6 @@
       },
       "login": {
         "description": "Commands to login using Digger API key"
-      },
-      "eject": {
```

Comment:
This seems like a merge fail, since these are hand written and describe dgctl commands for the help seciton

Reply:
updated it